src/main/java/com/laborhelper/enterprise/dto/LaborAction.java

```diff
@@ -5,15 +5,25 @@
 import java.util.Date;
 import java.util.Map;
 
+/*
+Maybe separate this class in multiple other classes?
+@nonNull is helpful but makes testing a little harder
+*/
 public @Data class LaborAction {
+    //Id Might want to be int if you aren't using letters in
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.